vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/VariantExporterControllerTest.java

```diff
@@ -116,82 +115,16 @@ public static void tearDownClass() throws UnknownHostException {
-        assertEquals(3, query.size());
-        assertEquals(studies, query.getAsStringList(VariantDBAdaptor.STUDIES));
-        assertEquals(files, query.getAsStringList(VariantDBAdaptor.FILES));
-        assertEquals(Arrays.asList("annotation", "sourceEntries.cohortStats"), query.getAsStringList("exclude"));
-
-
-        // some accepted parameters
-        MultivaluedMap<String, String> acceptedParameters = new MultivaluedHashMap<>();
-        String region1 = "1:1000-2000";
-        String region2 = "1:2500-3000";
-        acceptedParameters.add(VariantDBAdaptor.REGION, region1);
-        acceptedParameters.add(VariantDBAdaptor.REGION, region2);
-        String id = "rs1234";
-        acceptedParameters.add(VariantDBAdaptor.ID, id);
-        query = controller.getQuery(acceptedParameters);
-        assertEquals(5, query.size());
-        assertEquals(studies, query.getAsStringList(VariantDBAdaptor.STUDIES));
-        assertEquals(files, query.getAsStringList(VariantDBAdaptor.FILES));
-        assertEquals(Arrays.asList(region1, region2), query.getAsStringList(VariantDBAdaptor.REGION));
-        assertEquals(id, query.getString(VariantDBAdaptor.ID));
-        assertEquals(Arrays.asList("annotation", "sourceEntries.cohortStats"), query.getAsStringList("exclude"));
-
-
-        // mixed accepted and non accepted parameters
-        MultivaluedMap<String, String> mixedParameters = new MultivaluedHashMap<>();
-        mixedParameters.add(VariantDBAdaptor.REGION, region1);
-        mixedParameters.add(VariantDBAdaptor.REGION, region2);
-        mixedParameters.add(VariantDBAdaptor.ID, id);
-        mixedParameters.add("wrongParameter1", "1");
-        mixedParameters.add("wrongParameter1", "1b");
-        mixedParameters.add("wrongParameter2", "2");
-        query = controller.getQuery(mixedParameters);
-        assertEquals(5, query.size());
-        assertEquals(studies, query.getAsStringList(VariantDBAdaptor.STUDIES));
-        assertEquals(files, query.getAsStringList(VariantDBAdaptor.FILES));
-        assertEquals(Arrays.asList(region1, region2), query.getAsStringList(VariantDBAdaptor.REGION));
-        assertEquals(id, query.getString(VariantDBAdaptor.ID));
-        assertEquals(Arrays.asList("annotation", "sourceEntries.cohortStats"), query.getAsStringList("exclude"));
-
-    }
 
     @Test
     public void testVcfExportOneStudy()
             throws ClassNotFoundException, URISyntaxException, InstantiationException, IllegalAccessException,
-            IOException,
-            IllegalOpenCGACredentialsException {
+            IOException {
         String studyId = "7";
         List<String> studies = Collections.singletonList(studyId);
 
         VariantExporterController controller = new VariantExporterController(
-                TestDBRule.getTemporaryDBName(TestDBRule.HUMAN_TEST_DB),
+                TestDBRule.getTemporaryDBName(TestDBRule.HUMAN_TEST_DB), variantSourceService, variantService,
```

Comment:
The services are null, so this test is failing. Fixing TestDbRule should correct this.